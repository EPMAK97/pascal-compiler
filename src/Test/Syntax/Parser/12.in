var x : integer;
Begin
    x := 1;
    if x > 10 then
        x := 24;
end.
