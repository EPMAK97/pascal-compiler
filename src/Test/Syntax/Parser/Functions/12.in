program test;

function f(a, b, c: Integer) : Double;
begin
    Result := (a + b) / c;
end;

var

    r: Double;

begin
    r := f(1.0, 2);
end.