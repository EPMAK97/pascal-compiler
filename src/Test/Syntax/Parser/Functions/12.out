Error in pos 13:18 incompatible types: got DOUBLE expected INT
